src/commands/github-bot-comment-on-pr.yml

```diff
@@ -7,6 +7,13 @@ parameters:
   command:
     type: string
     description: A command which must produce a /tmp/notification.txt file. If no, or empty file, produced step is halted
+  update-last:
+    type: boolean
+    description: Update the last comment from this bot with this title instead of creating a new one
+    default: false
+  bot-making-comments:
```

Comment:
ideally this would have been mentioned as a new required parameter in changelog :)